rolling/distribution.yaml

```diff
@@ -2773,6 +2773,21 @@ repositories:
       url: https://github.com/DFKI-NI/mir_robot.git
       version: rolling
     status: developed
+  mobile_catkin_modules_build_development_tools:
+    doc:
+      type: git
+      url: https://github.com/Application-UI-UX/mobile_catkin_modules_build_development_tools.git
+      version: main
+    release:
+      tags:
+        release: release/rolling/{package}/{version}
+      url: git@github.com:Application-UI-UX/mobile_catkin_modules_build_development_tools.git
```

Comment:
This needs to be the https url not ssh. And should be a separate release repo.

Reply:
I know that it should be a separate release repo, when publishing it asked if i wanted to do it or not but it was not a hard requirement. And though that a separate branch under the same repo would suffice. Is the same repo good or is it a hard requirement?

Reply:
yea, great I have 2 organization that is in syncs, so I will do that instead. Like how the updated commits is.